csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,284 @@
+          },
+        },
+      },
+
+      // Table
+      tableCpuUtilization: {
+        name: 'tableCpuUtilization',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: '100 * sum by (instance_name, job, project_id)(stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{job!="",job=~".+",project_id=~".+"})',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableUptime: {
+        name: 'tableUptime',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(stackdriver_gce_instance_compute_googleapis_com_instance_uptime_total{job=~"$job", project_id=~"$project_id"})',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      tableSentPackets: {
+        name: 'tableSentPackets',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_packets_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableReceivedPackets: {
+        name: 'tableReceivedPackets',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_packets_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableSentBytes: {
+        name: 'tableSentBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableReceivedBytes: {
+        name: 'tableReceivedBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableReadBytes: {
+        name: 'tableReadBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
```

Comment:
In the table that I manually created [here](https://awsintegrationrevamp.grafana-dev.net/d/ddxod2rhnn474c/gcp-google-compute-engine?from=now-6h&to=now&timezone=browser&var-datasource=grafanacloud-prom&var-job=$__all&var-project_id=$__all&var-instance=gke-dev-eu-west-3-main-n2s8-1-b79cd1-bee11d00-54sh&showCategory=Override%207) I use `step: 1`, but I couldn't make this line work the same. Any idea if the prop is named differently?

Reply:
In this file, you're using the "Signals" interface, which is an abstraction on-top of the default Grafana schema. [Signals ](https://github.com/grafana/jsonnet-libs/blob/master/common-lib/common/signal/README.md)does not have any support for `step`, so you'll have to add it as an override in `panels.libsonnet`. I don't know that there is a good example of this particular override anywhere.

Reply:
Thank you both, this was really helpful!